index.js

```diff
@@ -51,7 +54,7 @@ function updateUi() {
 }
 
 function loadTopoJson(topoJson) {
-  clearInterval(cartogramComputeTimer)
+  cancelAnimationFrame(cartogramComputeRafId)
```

Comment:
i'm not positive this is still necessary... since the new RAFs are only called if the cartogram is `!iterated`.  you probably know better.